packages/landing/src/ol.attribution.ts

```diff
@@ -0,0 +1,127 @@
+import { AttributionCollection, AttributionStac } from '@basemaps/shared/build/attribution';
+import { StacLicense } from '@basemaps/shared/build/stac';
+import { View } from 'ol';
+import { Extent } from 'ol/extent';
+import OlMap from 'ol/Map';
+import MapEventType from 'ol/MapEventType';
+import Source from 'ol/source/Source';
+import { Attribution } from './attribution';
+import { MapOptions, MapOptionType, WindowUrl } from './url';
+
+const Copyright = `© ${StacLicense} LINZ`;
+
+async function fetchAttributions(url: string): Promise<AttributionStac | null> {
```

Comment:
Want to move this into the Attributions eg could do `Attributions.load(url: string)`